.github/pull_request_template.md

```diff
@@ -0,0 +1,2 @@
+Please replace this text with a summary of the change and which issue is fixed.
+You should mention the relevant JIRA issue like _ARTS-###_ which will be enable auto-linking the PR and JIRA.
```

Comment:
I think it is better to have a format  for the PR template... So all the PRs will be aligned on the structure too ... 

Something like: 

**Description**:
This PR fixes :  _ARTS-###_   ...
**Proposed changes**:..
**Screenshot** (optional)...
Not sure we need it here, but maybe a checklist before/after submitting , or main components involved...

Reply:
I expanded it a bit. Currently I don't have anything for checklist